cloud/cloud.go

```diff
@@ -8,10 +8,8 @@ import (
 	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/model/distro"
 	"github.com/evergreen-ci/evergreen/model/host"
-	"github.com/mongodb/grip"
-	"github.com/mongodb/grip/message"
 	"github.com/pkg/errors"
-	"go.mongodb.org/mongo-driver/bson"
+	"gopkg.in/mgo.v2/bson"
```

Comment:
Is this intentionally switched from mongo-driver/bson to mgo.v2/bson? We typically alias the latter to mgobson, and don't use unless absolutely necessary.

Reply:
Unintentional!